packages/dashmate/README.md

```diff
@@ -442,19 +463,17 @@ One of your nodes is not running, you may retry with the --force option:
 `dashmate group:stop --force` to stop group of nodes (local)
 
 #### Running services detected. Please ensure all services are stopped for this config before starting
-Some nodes are still running and preventing dashmate to make a proper start, that might be left after unsuccessful close. Try to stop them forcibly with --force option before trying to start
+Some nodes are still running and preventing dashmate from starting properly. This may occur after a command exits with an error. Try to stop force stop the nodes using the `--force` option before trying to run the `start` command again.
 
 `dashmate stop --force` to stop single node (fullnode / masternode)
 
 `dashmate group:stop --force` to stop group of nodes (local)
 
 #### externalIp option is not set in base config
-This may happen when you switch between multiple major versions, so your config became incompatible. In this case, do a manual reset and run setup again
+This may happen when you switch back and forth between major versions, making config incompatible. In this case, do a manual reset and run setup again
```

Comment:
```suggestion
This may happen when you switch back and forth between major versions, making config incompatible. In this case, do a manual reset and run setup again.
```